app/views/items/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<section class="page-wrapper">
+  <div class="page-title-bar">
+    <span class="page-title">Items</span>
+    <span>
+      <div class="dropdown">
+        <button class="dropbtn">Sort By</button>
+        <div class="dropdown-content">
+          <a href="?sort=max-price">Price: High - Low</a>
+          <a href="?sort=min-price">Price: Low - High</a>
+          <a href="?sort=a-z">Name: A - Z</a>
+          <a href="?sort=z-a">Name: Z - A</a>
+        </div>
```

Comment:
The way you're sorting these is good.. While you're using rails, I'd stick to using erb when you can. So what I mean by that is use a link that routes each option to the index page with those specific queries. 

```ruby 
<%= link_to "Price: High-Low", "/items?sort=max-price" %> 
```